xonsh/ansi_colors.py

```diff
@@ -3,6 +3,7 @@
 import sys
 import warnings
 import builtins
+import typing as tp
```

Comment:
It is probably more consistent to just `import typing` here

Reply:
it was taking lot of space. the Union and other complex notations will become long too. Do you want to replace yet?

Reply:
Well, yeah, so the issue is that the same abbreviation needs to be used everywhere and `tp` will enter the globals of the amalgamated module.  I am OK with `import typing as tp`, but mostly because this is something that then gets used globally in annotations elsewhere. Is this is standard way to abbreviate typing?

Reply:
I have seen it in some places in same vein as `numpy as np` . but definitely not the standard way. it is to reduce the visual Noise. If we keep it the same in all places, then I think we can add a flake8 rule.

Reply:
Yeah, it needs to be the same in all places, so a flake8 rule would be helpful.

Reply:
Ok